samples/star-wars-api/Startup.fs

```diff
@@ -31,7 +45,7 @@ type Startup private () =
             )
             // Use for pretty printing in logs
             .Configure<HttpClientJsonOptions>(
```

Comment:
And instead fo setting up options here we can have such options from our options.
I've made that in my rebased version of your branch